src/gui/src/stub_heatMap.cpp

```diff
@@ -195,4 +195,60 @@ double GlobalRoutingDataSource::getGridYSize() const
   return HeatMapDataSource::getGridYSize();
 }
 
+//////////
```

Comment:
Why the `//////////` ?

Reply:
I am simply complying with the style of the file that separates the implementation of each class with `//////////`